lib/galaxy/util/__init__.py

```diff
@@ -1627,6 +1627,18 @@ def url_get(base_url, auth=None, pathspec=None, params=None, max_retries=5, back
     return response.text
 
 
+def is_url(uri):
+    """
+    check if uri is (most likely) an url
+
+    >>> is_url('https://zenodo.org/record/4104428/files/UCSC-hg38-chr22-Coding-Exons.bed')
+    True
+    >>> is_url('/some/path')
+    False
+    """
+    return uri.find('://') != -1
```

Comment:
`file://` would be an allowed protocol, which I doubt is something we want to generally allow. I think we should use an allow list of protocols (could be a parameter to this function, defaulting to `http://` and `https://`).